CESC-submission/paper.tex

```diff
@@ -280,7 +277,7 @@ \section{Impact of Financial Incentives on Service Delivery}
 \paragraph{Discussion.} Which solution to adopt will be based on benchmark testing and the development vision of the NuCypher network. The ZKP-based solution (the one utilizing bulletproofs) provides several advantages including non-interactivity (as opposed to the committee-attestation one that needs interaction with the committee). It also supports computational soundness as opposed to the statistical one of the commit/challenge/open solution. Nonetheless, it could be the case that the other solutions are more efficient especially if we incorporate committee attestation or sidechains to support other functionalities in the network. 
 
 
-Another issue is regarding the payment computation. During early stages when the network launches, we anticipate a limited adoption and service demand. Thus, we reward Ursulas for just being online to encourage adoption. While in later stages, payments will be dispensed in proportion to the actual amount of service an Ursula provides. This is due to the fact that later on the inflation rewards will disappear, i.e., all tokens in the network will be minted, and the service fees are the only source of income. To incentivize Ursula to provide a correct and timely service, these rewards should be computed based on the actual amount of service. This shows how changes in incentive alignment lead to different views on how to distribute these incentives and ensure the security of the system.
+Another issue is regarding the payment computation. During early stages when the network launches, we anticipate a limited adoption and service demand. Thus, we reward Ursulas for just being online to encourage adoption. While in later stages, payments will be dispensed partially in proportion to the actual amount of service requests an Ursula answers. This is due to the fact that later on the subsidies will effectively disappear, i.e., the majority of tokens in the network will be minted, and the service fees are the only source of income. To incentivize Ursula to provide a correct and timely service, these fees should be computed based on the actual amount of service. This shows how changes in incentive alignment lead to different views on how to distribute these incentives and ensure the security of the system.
```

Comment:
While in later stages, payments will be dispensed *partially* in proportion to the actual amount of *service requests* an Ursula *answers*

*the majority* of tokens in the network will be minted